src/items/sxa-renderings/Project/PLAY/Page Content/HeroImage.yml

```diff
@@ -12,18 +12,18 @@ SharedFields:
   Value: "edgeicons/32x32/hero-image.png"
 - ID: "1a7c85e5-dc0b-490d-9187-bb1dbcb4c72f"
   Hint: Datasource Template
-  Value: /sitecore/templates/Project/PLAY/Legacy/HeroImage
+  Value: /sitecore/templates/Project/Legacy/HeroImage
 - ID: "a77e8568-1ab3-44f1-a664-b7c37ec7810d"
   Hint: Parameters Template
   Value: "{3C13BF6C-32DD-4F40-BCC9-208B47F21BB5}"
 - ID: "b5b27af1-25ef-405c-87ce-369b3a004016"
   Hint: Datasource Location
-  Value: ./Page Components|/sitecore/content/PLAY/playwebsite/Data/Components/HeroImage
+  Value: ./Page Components|/sitecore/content/PLAY/playwebsite/Data/Components/HeroImage|./Data
```

Comment:
If I understand the work done correctly, `./Page Components` should be replaced by `./Data` instead of adding `./Data` as a third option. Is that right?

Check the other rendering items as well as I think there are remaining `./Page Components` datasource locations that should be replaced by `./Data`.